autogpt_platform/backend/backend/blocks/airtable/_api.py

```diff
@@ -1037,16 +1032,14 @@ def oauth_authorize(
     request_params = OAuthAuthorizeRequest(
         client_id=client_id,
         redirect_uri=redirect_uri,
-        response_type=response_type,
         scope=" ".join(scopes),
         state=state,
         code_challenge=code_challenge,
-        code_challenge_method=code_challenge_method,
     )
 
     # Build the authorization URL
     base_url = "https://airtable.com/oauth2/v1/authorize"
-    query_string = urlencode(request_params.model_dump())
+    query_string = urlencode(request_params.model_dump(exclude_none=True))
```

Comment:
None of the attributes of `OAuthAuthorizeRequest` are allowed to be `None` so this seems unnecessary (same below for `OAuthTokenRequest`)

Reply:
agreed 👍